setup.py

```diff
@@ -55,4 +56,13 @@
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     packages=find_packages(exclude=('tests',)),
```

Comment:
Might need package_data or MANIFEST.in or both to include the .pyx file in the package. I'm not a python packaging expert so not sure

Reply:
The updated setup.py takes care of the inclusion.